tests/testthat/test-stat-bin.R

```diff
@@ -22,6 +22,14 @@ test_that("bins specifies the number of bins", {
   expect_equal(nrow(out(bins = 100)), 100)
 })
 
+test_that("binwidth computes widths for function input", {
+  df <- data.frame(x = 1:100)
+  out <- function(x, ...) {
```

Comment:
Ok, I get the simpler function that just returns a constant, but I don't know what "indirection of this function" means.